tests/integration/test_alert_rules_load.py

```diff
@@ -16,56 +16,38 @@
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 app_name = METADATA["name"]
 resources = {"loki-image": METADATA["resources"]["loki-image"]["upstream-source"]}
-rules_app = "cos-config"
-rules_app2 = "cos-config2"
-rules_app3 = "cos-config3"
+rules_app = "loki-tester"
```

Comment:
As a general comment, we should just `git rm` this entire file. It's obvious from the changes below and from running it that there are no meaningful differences between this and `test_multiple_rule_providing_apps`

Reply:
I agree. But I thought it would be cheating to remove tests as a way to demonstrate improvements in stability of integration tests in a PR trying to do so.